### DISCUSSION TITLE
How do You right click?

### DISCUSSION THREAD

**Discussion by Wolvesman** [spyoungtech/ahk#160] [Q&A] [answered]
I Have tried various combinations including:
ahk.key_down('shift')
time.sleep(1)
ahk.key_down('{F10}')
time.sleep(1)
ahk.key_up('{F10}')
time.sleep(1)
ahk.key_up('shift') 

The shift key is actvating on the OSK app on windows 10 but not F10.

Thanks for your help.

**Comment by spyoungtech** [comment, marked as answer]
I see. Yeah, I don't think `key_up` and `key_down` work with `{F10}` like that because of how these work... The resulting script would be 
```ahk
SendInput {{F10} up}
```
Which would not be correct.

I think you want to do is `ahk.key_down('F10')`

The resulting script looks like this:

```ahk
SendInput {F10 up}
```
Which is what you want, and should work as expected

> How do You right click?

```python
ahk.right_click()
```

**Comment by spyoungtech** [comment]
@Wolvesman let me know if you still need help with this. Otherwise, I'll consider the issue resolved.

**Comment by spyoungtech** [comment]
This is now no longer be an issue since v1
